projects/spectator/src/lib/is-type.ts

```diff
@@ -0,0 +1,5 @@
+import { Type } from '@angular/core';
+
+export function isType(v: any): v is Type<any> {
```

Comment:
`isFunction`?

Reply:
I actually copied this type guard from Angular: https://github.com/angular/angular/blob/4c2ce4e8ba4c5ac5ce8754d67bc6603eaad4564a/packages/core/src/type.ts

Reply:
Not sure what is better. Let me know if you insist of `isFunction`. "Is function" is what the implementation does, but it's intention is to check whether you pass a Type (Component). Also, it is a _type guard_ for `Type`.

Reply:
Ok, let's leave it.